docs/docs/sourcing-from-drupal.md

```diff
@@ -2,7 +2,48 @@
 title: Sourcing from Drupal
 ---
 
-This is a stub. Help our community expand it.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Why use Drupal + Gatsby together?
+
+Using Drupal as a headless CMS with Gatsby is a great way to get an enterprise-quality CMS for free, paired with a great modern development experience and all the benefits of the JAMstack, like performance, scalability, and security.
+
+It only takes a few steps to use Gatsby with Drupal as a headless CMS (also known as decoupled Drupal).
+
+<a href="https://using-drupal.gatsbyjs.org/">A delicious Gatsby + Drupal site</a>
+
+### How to implement Drupal + Gatsby
+
+Hooking up Gatsby to a new or existing Drupal site takes just a few steps:
+
+- Add the `gatsby-source-drupal` plugin to your Gatsby site
+- Configure the plugin with your Drupal credentials & hosted URL
+- Access all your Drupal data using GraphQL queries
+
+### Why use Gatsby and Drupal together?
+
+<a href="https://www.acquia.com/drupal/decoupled-drupal">“Decoupled Drupal”</a> has become an increasingly popular approach to building enterprise-grade websites, and has the <a href="https://dri.es/how-to-decouple-drupal-in-2018">full-throated support</a> of Drupal community leaders. Using Gatsby in a decoupled Drupal setup allows your team to access the powerful content modeling and access workflow capabilities of Drupal 8, as well as the powerful UI creation & performance toolset of Gatsby.
+
+### When is Drupal a great choice?
+
+Many development teams, content teams, and client decision-makers are familiar with Drupal. Here are some scenarios in which Drupal is a great choice (and a few scenarios for which it’s not-so-great):
+
+### Drupal is great for:
+
+- Complex page layouts or content modelling with multiple sections per page
```

Comment:
I think it's `modeling` 🙃 But apparently varies between US + everywhere else, ha.